### PR TITLE
49 hotfix summary throughput

### PR DESCRIPTION
**Pull request by grsterin**

### CODE REVIEW

File: database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala [hunk 1]
```diff
@@ -23,9 +23,9 @@ class MetadataDataAccessComponent(val driver: JdbcProfile) extends DataAccessCom
   val metadataEntriesToSummarizeQuery = {
     Compiled(
       (limit: ConstColumn[Long]) => (for {
-        (_, metadataEntry) <-
-          summaryQueueEntries join metadataEntries on (_.metadataJournalId === _.metadataEntryId)
-      } yield metadataEntry).sortBy(_.metadataEntryId).take(limit)
+        summaryEntry <- summaryQueueEntries.take(limit)
+        metadataEntry <- metadataEntries if metadataEntry.metadataEntryId === summaryEntry.metadataJournalId
```
Comment (cjllanwarne) [inline database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala]: might this now need to make 50,000 queries to the database (one per summary queue entry) which is taken?

Comment (grsterin) [reply database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala]: No, here's the generated query:
```
select x2.`WORKFLOW_EXECUTION_UUID`, x2.`CALL_FQN`, x2.`JOB_SCATTER_INDEX`, x2.`JOB_RETRY_ATTEMPT`, x2.`METADATA_KEY`, x2.`METADATA_VALUE`, x2.`METADATA_VALUE_TYPE`, x2.`METADATA_TIMESTAMP`, x2.`METADATA_JOURNAL_ID` from (select `METADATA_JOURNAL_ID` as x3 from `SUMMARY_QUEUE_ENTRY` limit ?) x4, `METADATA_ENTRY` x2 where x2.`METADATA_JOURNAL_ID` = x4.x3 order by x2.`METADATA_JOURNAL_ID`
```

Comment (cjllanwarne) [reply database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala]: ok - great!

File: database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala [hunk 2]
```diff
@@ -207,8 +207,8 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
         )
       summarizedMetadataEntryIds = rawMetadataEntries.flatMap(_.metadataEntryId)
       _ <- deleteSummaryQueueEntriesByMetadataJournalIds(summarizedMetadataEntryIds)
-      unsummarizedTotal <- countSummaryQueueEntries()
-    } yield (summarizedMetadataEntryIds.length.toLong, unsummarizedTotal.toLong)
+      //unsummarizedTotal <- countSummaryQueueEntries()
+    } yield (summarizedMetadataEntryIds.length.toLong, 42L /*unsummarizedTotal.toLong*/)
```
Comment (cjllanwarne) [inline database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala]: Do we want a JIRA ticket to solve this for next release?

Comment (cjllanwarne) [reply database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala]: I'll make one

**Comment by cjllanwarne** [comment]
Reminder: could you also make a non-hotfix version of this PR (leaving out the metrics change for now, since that's going to be covered separately/properly in https://broadworkbench.atlassian.net/browse/BA-6307)

**Pull request merged**